app/include/dt-bindings/zmk/keys.h

```diff
@@ -146,4 +146,5 @@
 #define MOD_RCTL (1 << 0x04)
 #define MOD_RSFT (1 << 0x05)
 #define MOD_RALT (1 << 0x06)
-#define MOD_RGUI (1 << 0x07)
\ No newline at end of file
+#define MOD_RGUI (1 << 0x07)
+#define MOD_LAST_MODIFIER MOD_RGUI
```

Comment:
What was `MOD_LAST_MODIFIER` for? I don't see it used anywhere.

Reply:
I used it before but didn't need it later. removed.